corehq/apps/app_manager/templates/app_manager/partials/bulk_upload_app_translations.html

```diff
@@ -17,5 +17,15 @@
         </select>
       </p>
     </li>
+    {% if request|toggle_enabled:'APP_TRANSLATIONS_WITH_TRANSIFEX' %}
+    <li>
+      <p>
+        {% trans "Download only translations eligible for Transifex?" %}
```

Comment:
nit: do we need the '?' in the end?

Reply:
Probably pedantic, but when a checkbox is used to select options, then there wouldn't be a question mark because the UI is telling you to choose something. But when you are using a checkbox to answer a question (essentially "Do you want to download only translations eligible for Transifex?" "Yes" or "No") then the question should end in a question mark. I think if I were to change this, it would be to keep the question mark, but replace the checkbox with a yes-no toggle/switch.